Scoop/Supporting/Utilities/ModalViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ModalViewController.swift
+//  Scoop
+//
+//  Created by Richie Sun on 10/18/23.
+//
+
+import UIKit
+
+class ModalViewController: UIViewController {
+
+    // MARK: - Views
+
+    let contentView = UIView()
```

Comment:
Should this be private?

Reply:
The ModalViewController is a template class for all modal Views in scooped, so I created these functions to be used to setup the modal view. If I make these private, then a VC that is a subclass of ModalVC is unable to access the function to setup the modal tab, and you need the content view to be able to add things to the view